yarn.lock

```diff
@@ -135,6 +135,42 @@
   dependencies:
     any-observable "^0.3.0"
 
+"@sinonjs/commons@^1", "@sinonjs/commons@^1.6.0", "@sinonjs/commons@^1.7.0", "@sinonjs/commons@^1.7.2":
```

Comment:
These changes should be in spec-main/yarn.lock

Reply:
Moved in 839eebe